### ISSUE TITLE
Replace SceneSummaryNode with ScreenSummaryNode

### ISSUE THREAD

**Issue by zepumph** [phetsims/scenery-phet#393]
`SceneSummaryNode` is now deprecated, and `ScreenSummaryNode` is the way of the future. We should change all usages in the project to use this new strategy. This includes custom things like BASE's own "SceneSummaryNode"

Adding chip away and assigning to everyone.

Here are the places where SceneSummaryNode is used:

- [x] Ohms Law
- [x] Rutherford Scatterting
- [x] BASE (uses its own implementation of a SceneSummaryNode

Here are the wildcard places that are just using a non uniform pattern and just have their own "Scene Summary" string:
- [x] JT
- [x] MAL
- [x] RIAW
- [x] FL

Once all usages above have been taken care of:
- [ ] `addScreenSummaryNode` option should be removed from `ScreenView` and all sims should opt into having a screenSummaryNode by default.
- [x] Delete SceneSummaryNode from scenery-phet.
- [x] make sure `sceneSummarySingleScreenIntro` is deleted from `SceneryPhetA11yStrings`.
- [x] Delete sceneSummary from SceneryPhetA11yStrings?

**Comment by zepumph** [comment]
Tagging for meeting to make sure we are all on the same page.

**Comment by jessegreenberg** [comment]
I went through the sims, all are using `screenSummaryNode`. We can proceed with the last four check boxes but maybe we should wait until #381 is discussed.

**Comment by jessegreenberg** [comment]
The last check box
> addScreenSummaryNode option should be removed from ScreenView and all sims should opt into having a screenSummaryNode by default.

Will need to be done as part of https://github.com/phetsims/scenery-phet/issues/381. But I think this issue is complete, and the project no longer has any usages of sceneSummary or references to 'scene summary'. I am going to close this issue, but pinging @zepumph in case he feels there is more to do.
